pkg/antctl/raw/clusterset/command.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusterset
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var (
+	Command *cobra.Command
+	option  = &struct {
+		namespace    string
+		outputFormat string
+	}{}
+)
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "clusterset",
+		Aliases: []string{"clustersets"},
+		Short:   "Print Multi-cluster ClusterSets",
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 0 {
+				_ = runE(cmd, args[0])
+			} else {
+				_ = runE(cmd, "")
+			}
+		},
```

Comment:
Maybe it is better to put clusterID argument in options struct and use flags to get it .